lib/services/eula-service-base.ts

```diff
@@ -60,7 +61,7 @@ export abstract class EulaServiceBase implements IEulaService {
 
 		// At this point we should have already downloaded the EULA, so just get the info for local file.
 		// If it does not exist - we were unable to download it.
-		const currentEulaHash = await this.getLocalEulaHash();
+		const currentEulaHash = await this.$nsCloudHashService.getLocalFileHash(this.getPathToEula());
```

Comment:
Not a merge-stopper, but instead of this approach you could change the `getLocalEulaHash` implementation to 
```TypeScript
private async getLocalEulaHash(): Promise<string> {
	return this.$nsCloudHashService.getLocalFileHash(this.getPathToEula());
}
```

And leave both calls unaltered